frontend/src/pages/Report.tsx

```diff
@@ -29,6 +29,8 @@ import { AuthContext } from "../components/AuthProvider";
 import { useParams } from "react-router-dom";
 import ClimbingBoxLoader from "react-spinners/ClimbingBoxLoader";
 import LoadingOverlay from "react-loading-overlay-ts";
+import warning from "../icons/exclamation-triangle.svg";
```

Comment:
Why not use the same as in the unsaved changes warning?

Reply:
Actually, the one in the unsaved changes warning isn't a real image but just a symbol in a p tag. So in this case, I'd prefer the image because it is more semantic and can have a proper alt tag, title etc.